README.md

```diff
@@ -2,8 +2,9 @@
 <!-- markdownlint-disable MD033 -->
 
 ![Build status](https://badge.buildkite.com/df9e995b3a5e4b0bebce8b432b0bf48b092fd261b7017b65c1.svg)
-[![License](https://img.shields.io/github/license/opstrace/opstrace)](https://github.com/opstrace/opstrace/blob/main/LICENSE)
+[![License](https://img.shields.io/github/license/opstrace/opstrace)](LICENSE)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
+[![Calver YYYY.0M.0D](https://img.shields.io/badge/calver-YYYY.0M.0D-22bfda.svg)](https://calver.org)
```

Comment:
How do we keep the value of the badge up to date?

Reply:
Manually.  The intent of this PR was to propose we see this badge as a way to direct people towards releases.  We could leave the pattern we've chosen there, or we could update it with the latest release.

If we can't decide what to do here, we can certainly remove it for now.

Reply:
Unless this badge is automated, we should remove it - it'll be too easy to forget about updating that little part, besides the release shows up on the right sidebar in Github